lib/Doctrine/Annotations/Metadata/PropertyMetadata.php

```diff
@@ -22,12 +22,12 @@ final class PropertyMetadata
     /** @var bool */
     private $default;
 
-    /** @var array<int|float|string|bool>|null */
+    /** @var array<string, array<int|float|string|bool>|string>|null $enum */
```

Comment:
One more nitpick: properties should not include `$propernyName`.

Reply:
Good catch! Damn copy/paste!